samples/rawvm/k8cli.yaml.in

```diff
@@ -3,6 +3,8 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: raw-cli-deployement
+  labels:
+    app: fortio
```

Comment:
mismatch on name and app

Reply:
Removed based on https://github.com/istio/istio/pull/27242#issuecomment-692549416